Source/Scene/PostProcessorScene.js

```diff
@@ -20,23 +20,247 @@ define([
      */
     function PostProcessorScene() {
         this.blackAndWhiteStage = createBlackAndWhiteStage();
+        this.blackAndWhiteStage.show = false;
         this.brightnessStage = createBrightnessStage();
+        this.brightnessStage.show = false;
         this.eightBitStage = createEightBitStage();
+        this.eightBitStage.show = false;
         this.compositeTextureStage = createCompositeTextureStage();
         this.compositeTextureStage.show = false;
+        this.LensFlareStage = createLensFlareStage();
 
         var stages = [
             this.blackAndWhiteStage,
             this.brightnessStage,
             this.eightBitStage,
-            this.compositeTextureStage
+            this.compositeTextureStage,
+            this.LensFlareStage
         ];
 
         this._postProcessor = new PostProcessor({
             stages : stages
         });
     }
 
+    function createLensFlareStage() {
+        var url_Dirt = buildModuleUrl('Assets/Textures/LensFlare/DirtMask.jpg');
```

Comment:
For local variable naming conventions, we use `camelCase`.

If you haven't already browse through our [Coding Guide](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide).